samples/bluetooth/direct_test_mode/src/fem/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+DT_COMPAT_NORDIC_NRF21540 := nordic,nrf21540-fem
+
+config NRF21540_FEM
+	bool "nRF21540 Front-End-Module"
+	default y if $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540))
```

Comment:
Does nrf21540_FEM work without DT_COMPAT_NORDIC_NRF2154? If not it should be:
```
depends on $(dt_nodelabel_has_compat,nrf_radio_fem,$(DT_COMPAT_NORDIC_NRF21540))
default y
```

Reply:
Right. Done